atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/CellsSweeper.java

```diff
@@ -55,15 +55,23 @@ public CellsSweeper(
             TransactionManager txManager,
             KeyValueService keyValueService,
             Collection<Follower> followers) {
+        this(txManager, keyValueService, followers, AtlasDbConstants.DEFAULT_INITIALIZE_ASYNC);
+    }
+
+    public CellsSweeper(
+            TransactionManager txManager,
```

Comment:
The indentation between this function and the above function is different. This will probably break checkstyle.